desktop/apps/artwork/client/index.coffee

```diff
@@ -72,9 +73,20 @@ module.exports =
           require('../../../components/current_auctions/index.jsx').default unless context.is_closed
           require '../components/artist_artworks/index.coffee' if context.is_closed
           require '../components/related_artworks/index.coffee' if context.is_closed
-          require '../components/related_artists/index.coffee'
+          require '../components/related_artists/index.coffee',
         ]
 
+      # Make some Auctions-only layout tweaks. See https://github.com/artsy/auctions/issues/360#event-1017559326
+      # for more info.
+      postInit: ->
+        $currentAuctions = $ '.artwork-current-auctions'
+        $relatedRail = $ '.artwork-artist-related-rail'
+        paddingTop = if $currentAuctions.length then '42px' else '31px'
+
+        $relatedRail
+          .css('border-top', 0)
+          .css('padding-top', paddingTop)
```

Comment:
👍 - Could you maybe file this as an issue in Force? If you think it's a priority, we should loop back and refactor the entire artwork section all at once with some concerted attention so that comments like this don't get lost within tiny PR's. `/artwork` has a lot of moving pieces and a lot of different people worked on it. For right now, one thing at a time 😄. 

Are you going to open a new PR with your changes? If your tweaks came from the Chrome's devtools there might be some drift as you edit the `.styl` files.